ingest/src/main/java/com/microsoft/azure/kusto/ingest/StreamingIngestClient.java

```diff
@@ -120,13 +134,19 @@ public IngestionResult ingestFromStream(StreamSourceInfo streamSourceInfo, Inges
             throw new IngestionClientException(String.format("Mapping reference must be specified for DataFormat '%s' in streaming ingestion.", dataFormat.name()));
         }
 
+        ClientRequestProperties clientRequestProperties = null;
+        if (clientRequestId != null) {
```

Comment:
nit: Currently this either has a real value or `null`, but I think it's future-safer to instead test `StringUtils.isBlank()`.